examples/fedopt_example/server.py

```diff
@@ -53,20 +53,8 @@ def metric_aggregation(all_client_metrics: List[Tuple[int, Metrics]]) -> Metrics
     return server_metrics.compute_metrics()
 
 
-def fit_metrics_aggregation_fn(all_client_metrics: List[Tuple[int, Metrics]]) -> Metrics:
```

Comment:
These functions were redundant, we might as well just use the `metric_aggregation` function rather than having both of these "encapsulations"